tests/test_databases.py

```
@@ -451,11 +639,50 @@ def test_aws_extract_rds_storage_size():
+        if region == "ap-southeast-7":
+            raise ConnectionError(f"rds.{region}.amazonaws.com refused")
+        if region == "us-east-1":
+            return options
+        return []
+
+    @contextmanager
+    def capture(vendor, on_error=None):
+        try:
+            yield
+        except Exception:
+            pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)